my_price_scout/files/app_logic.py

```diff
@@ -263,14 +266,14 @@ def change_product_links(self):
 
         self.product = self.user.get_item(name)
 
+
         print("Which website's link would you like to remove? Copy and paste the website and link that is currently being tracked from above.")
         website, url = self.user_inputs.capture_website()
 
         self.specific_product = self.add_specific_product()
 
-        self.product.change_url_for_specific_product(
-            self.specific_product, website, url)
-        self.save_user()
```

Comment:
this will make user to not update in db after changing the link and we need that to happen